src/components/cylc/Toolbar.vue

```diff
@@ -0,0 +1,150 @@
+        </v-btn>
+        <!-- title -->
+        <span class="c-toolbar-title">{{ title }}</span>
+      </v-toolbar-title>
+    </div>
+
+    <!-- control bar elements displayed only when a suite has been positioned -->
+    <template v-if="tree && tree.length > 0">
+      <a @click="onClickPause">
+        <v-icon color="#5E5E5E">mdi-pause</v-icon>
+      </a>
+
+      <a @click="onClickStop">
+        <v-icon color="#5E5E5E">mdi-stop</v-icon>
+      </a>
+
+      <a>
+        <v-chip color="#E7E7E7" @click="toggleExtended">Control</v-chip>
+      </a>
+
+      <span>Running, will stop at 30000101T0000 cycle</span>
+
+      <v-spacer />
+
+      <a class="add-view" @click="onClickAddView">
+        Add View <v-icon color="#5995EB">mdi-plus-circle</v-icon>
+      </a>
+    </template>
+
+    <!-- displayed only when extended===true -->
+    <template v-slot:extension v-if="extended">
+      <span style="margin-left: 260px;">
+        <a @click="onClickPause">
+          <v-icon color="#5E5E5E">mdi-pause</v-icon>
+        </a>
+
+        <a @click="onClickStop">
+          <v-icon color="#5E5E5E">mdi-stop</v-icon>
+        </a>
+
+        <span>Other controls added in the future</span>
+      </span>
+    </template>
+
+  </v-toolbar>
+</template>
+
+<script>
+
+  import { mapMutations, mapState } from 'vuex'
+
+  export default {
+  data: () => ({
+    responsive: false,
+    responsiveInput: false,
+    extended: false
+  }),
+
+  computed: {
+    ...mapState('app', ['title']),
+    ...mapState('suites', ['tree'])
+  },
+
+  mounted () {
+    this.onResponsiveInverted()
+    window.addEventListener('resize', this.onResponsiveInverted)
+  },
+  beforeDestroy () {
+    window.removeEventListener('resize', this.onResponsiveInverted)
+  },
+
+  methods: {
+    ...mapMutations('app', ['setDrawer', 'toggleDrawer']),
+    onClickBtn () {
+      this.setDrawer(!this.$store.state.app.drawer)
+    },
+    onResponsiveInverted () {
+      if (window.innerWidth < 991) {
+        this.responsive = true
+        this.responsiveInput = false
+      } else {
+        this.responsive = false
+        this.responsiveInput = true
+      }
+    },
+    onClickPause () {
+      console.log("Pausing workflows has not been implemented yet")
+    },
+    onClickStop () {
+      console.log("Stopping workflows has not been implemented yet")
+    },
+    toggleExtended () {
+      this.extended = !this.extended
+    },
+    onClickAddView () {
+      console.log("Adding views has not been implemented yet")
+    }
+  }
+}
+</script>
+
+<style lang="scss">
+  .v-toolbar {
+    min-height: inherit !important;
+    .v-toolbar__content {
+      min-height: inherit !important;
+    }
+  }
+
+  .c-toolbar {
+    background-color: #E7E7E7;
+    border-bottom: 3px solid #ccc;
+
+    .c-toolbar-title, .add-view {
+      font-size: 22px !important;
+      color: #5995EB;
+      font-weight: 500;
+    }
+
+    .c-toolbar-title {
+      padding-right: 13px;
```

Comment:
Set in `em` so that it adjusts automatically with `font-size`?

Reply:
I normally use em... but the vuetify theme is using px (even for font size?).

I believe the left padding was something like 26px.

So I just used half its size for the right padding.

Reply:
> but the vuetify theme is using px (even for font size?).

What! That's horrible, it's hard-coded everywhere? What about accessibility.

Reply:
I think people are now going back from `em` to `px` for the finest of fine control.

Reply:
Which means universally changing font size becomes practically impossible...

Reply:
>I think people are now going back from em to px for the finest of fine control.

Interesting. When I learned about `em` (some years ago) I remember it was considered good practice to use it instead of `px`. But now that you mention it, I've seen other components in Vue.js that use `px` instead.

I think if you use `@media` in CSS, you can have different height/font size/etc... meaning that you don't really need `em`. So maybe that's the case in Vuetify? They actually have an issue that I think is related to this: [[Feature Request] Use rem, em instead of px](https://github.com/vuetifyjs/vuetify/issues/3401)

Reply:
(said that, I think using `px` for now should be OK, as the whole app is using `px` in Vuetify, and if we use `em` here later Vuetify could be using `rem` instead... so easier to follow that discussion and use the same as the underlying theme/framework IMHO)

Reply:
OK.

BTW my standard approach is along the lines of:

```css
@media only screen and (max-width: x px) {
    font-size: 10px;
}
@media only screen and (max-width: y px) {
    font-size: 12px;
}
@media only screen and (max-width: z px) {
    font-size: 15px;
}
h1 {
    font-size: 2em;
}
h2 {
    font-size: 1.6em;
}
/* etc ... */
```

Reply:
I like this one too, we can aim at having something similar to this :+1: